cheroot/testing.py

```diff
@@ -21,10 +22,19 @@
 ANY_INTERFACE_IPV4 = '0.0.0.0'
 ANY_INTERFACE_IPV6 = '::'
 
+
+def my_crazy_app(environ, start_response):
```

Comment:
I'm not sure that it belongs here. How can this be useful for testing apps by end-users?